### DISCUSSION TITLE
Can you tell me the history and the advantages and disadvantages of BiblioteQ?

### DISCUSSION THREAD

**Discussion by myunggg** [textbrowser/biblioteq#213] [Q&A] [answered]
Can you tell me the History and What are the advantage and disadvantage of BiblioteQ?

**Comment by meteos77** [comment]
Hello, from a user's point of view
- a creator who listens to the users; thank you textbrowser
- a very stable and multi-OS application

**Comment by textbrowser** [comment, marked as answer]
History

The original project was XBook and it was started in 2002 for Solaris. Think X11/Motif. XBook was renamed to BiblioteQ sometime in 2006.

Advantages

You'll have to determine those. BQ is available almost everywhere. From PowerPC to Sparc to ARM. It doesn't require a network. It supports two database engines and would support others with some tweaking of the SQL queries.

Disadvantages

You'll also have to determine those. Perhaps more integration with popular download sites, but this can be a lot of work. Also, more customization. Also lots of work. Only a single developer. Once I expire, the project will indubitably expire as well. :)
